### ISSUE TITLE
v2.12.3: "Share into" launcher not working

### ISSUE THREAD

**Issue by anlar** [gsantner/markor#2371]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

After update from v2.12.2 to v2.12.3 "Share into" launcher become broken. I click on it, quick animation of its trying to open screen appears, and then nothing happens.

Reproduced in both Fdroid version and nightly build.

### Steps to reproduce

1. In Markor settings enable "Launcher" option.
2. Add "Share into" launcher to home screen.
3. Open "Share into".

AR: quick animation appears, but window no opened.
ER: open share into screen.

### Information

Android version: 14
Device: Samsung Galaxy A55
App Version: 2.12.3
Nightly App Version: net.gsantner.markor-v153-2.12.4-flavorAtest-debug.apk

### Source

F-Droid

### Format / File type

Not specific

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment]
Can you please attach a debug log (logcat)?

I cannot reproduce, and I use ShareInto daily in both nightly and 
release version, with success.

**Comment by gsantner** [comment]
Seems like some buggy WebView at firdt sight, at least no such crash here.

Does the editor/viewer work for you fine?

**Comment by anlar** [comment]
Yes, editor and viewer screens works fine in Markor itself (for both todo and markdown files).

Attached video with issue.

https://github.com/user-attachments/assets/10c32756-c077-497e-aa43-da47571b04a8

**Comment by SebJM** [comment]
Bump. I have the same exact issue to the letter. My favourite feature has gone astray! 

In case it helps I'm using a Samsung Galaxy S21 Ultra: One UI version 6.1, Android Version 14, using patch level: 1 May 2024.

**Comment by gsantner** [comment]
Thank you for the video. This is why describing issues best you can helps a lot - especially if you add picture or video.

I thought you generally speak about ShareInto (when you share....something from browser to Markor) i.e.

I see that it's crashing here too. Probably related to @harshad1 recent changes to the activity launchers

**Comment by gsantner** [comment]
Thank you for the video. This is why describing issues best you can is important. It helps a lot if you add picture or video, makes it more likely to understand what you mean.

I thought you generally speak about ShareInto (when you share....something from browser to Markor) i.e.

I see that it's crashing here too. Probably related to @harshad1 recent changes to the activity launchers

**Comment by TruongNguyen5395** [comment]
I experienced this issue too in my Z fold 5, One UI

**Comment by e-ginger** [comment]
Same bug fixed. "Share into" launcher shortcut crashed, when I launch it from app menu or from widget.
Device Samsung Galaxy Note 10 (SM-N970F).
One UI 4.1
Android 12

In the newest Samsung Galaxy S25 this crashed too.
One UI 7.0
Android 15

**Comment by anlar** [comment]
Checked with latest version (v2.15.2) - still crashes.
